### DISCUSSION TITLE
2021.03.04 목 회의록

### DISCUSSION THREAD

**Discussion by hakju** [hakju/ios-cloud-notes#38] [General]
1. STEP 2 PR보내기
2. TIL 작성하기 (제발 쓰자ㅠㅠ)
3. Cloud 예습 (clone coding)
4. 늦어도 12시에는 맥북을 끄기
5. animator clone coding & frame과 bounds의 차이 알아보기

**Comment by soleJin** [comment]
+ step2 PR 보냄
+ 12시에 맥북끄기
+ TIL
